tests/unit/service/test_online_enable.py

```diff
@@ -63,4 +82,34 @@ async def test_update_feature(
         feature=online_disabled_feat,
         expected_online_enabled=False,
         expected_online_enabled_feature_ids=[],
+        expected_number_of_feature_lists=1,
+    )
+
+
+@pytest.mark.asyncio
+async def test_update_feature_when_already_enabled(
```

Comment:
Added test case here. The other test changes are due to changes in payload fixtures.